File: app/src/main/java/com/happs/ximand/ringcontrol/model/object/command/simple/ChangeManualModeCommand.java
package com.happs.ximand.ringcontrol.model.object.command.simple;

public class ChangeManualModeCommand extends SimpleBluetoothCommand {

    public static final byte COMMAND_CODE = 22;

    public ChangeManualModeCommand(boolean value) {
        super(COMMAND_CODE, ((byte) (value ? 0xff : 0)));
    }

}

File: app/src/main/java/com/happs/ximand/ringcontrol/model/object/command/simple/ChangeManualModeCommand.kt
package com.happs.ximand.ringcontrol.model.`object`.command.simple

class ChangeManualModeCommand(value: Boolean) :
        SimpleBluetoothCommand(CODE, (if (value) 0xff else 0).toByte()) {

    companion object {
        const val CODE: Byte = 22
    }
}